packages/formatter/src/date.ts

```diff
@@ -217,11 +219,15 @@ export function formatDateWithinRange(
  * @param date The date to format.
  * @param timezone The time zone to use for formatting.
  * @param baseDate The base date to use for comparison.
+ * @param maxPrecision Never show smaller units than the max precision.
```

Comment:
Nit: This removes the double-negative in the comment
```suggestion
 * @param maxPrecision Only show units as large or larger than the max precision.
```